CITi-UFPE/CITiMarco
# Feature/google login

## Whats I did
- Added login and logout components to the project (not working correctly yet)

## How to test 
1. yarn start
1. Use the enviroment variable I sent  on WhatsApp
1. Check if the login with google appears on the screen
1. Make the login with your google account and check if appears an alert with your name (if it doen't work try to clear your bworser cache data, as described: Chrome: Settings → Advanced → Clear browsing data → Cached images and files)

Resolves #8
